api/client/options.proto

```diff
@@ -41,6 +42,8 @@ message CommandLineOptions {
     // See :option:`--request_options` for details.
     RequestOptions request_options = 12;
   }
+  // See :option:`--cluster_config` for details.
+  envoy.api.v2.Cluster cluster_config = 13;
```

Comment:
Yeah, the basic approach is good, but I'm wondering what we can say about which parts of the `Cluster` config will be honored vs. ignored?